### ISSUE TITLE
Incorrect messages for problems opening database

### ISSUE THREAD

**Issue by itagagaki** [1Remote/1Remote#792]
If the database does not provide write permission, I believe the message key `string_database_error_permission_denied` (row 108 in glossary.csv) would be used. However, the message text in each language corresponding to this key seems to say “no read permission” or something similar.

It may also be necessary to check if this key is used in other cases other than no write permission.

Also recheck row 107 (`system_options_data_security_error_can_not_open`).

**Comment by VShawn** [comment]
You are right, it would be better to change `string_database_error_permission_denied` to "no read or write permission" and similar.

Additionally `system_options_data_security_error_can_not_open` has been deprecated, which was previously used as the prompt when an encrypted SQLite database was opened with a wrong password. This sentence will only be useful if database encryption feature is added back in the future. https://github.com/1Remote/1Remote/issues/755

**Comment by VShawn** [comment]
i confirmed here is the only reference of `string_database_error_permission_denied`

https://github.com/1Remote/1Remote/blob/main/Ui/View/Settings/DataSource/SqliteSettingViewModel.cs#L111-L115
